aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1841,7 +1798,9 @@ static Tensor& linalg_norm_out_impl(Tensor& result, const Tensor& self, optional
     // 'ord' is int or None
     std::vector<int64_t> dim_ = opt_dim.has_value() ? opt_dim.value().vec() : make_dim_list(ndim);
     if (!opt_num_ord.has_value() || dim_.size() == 1) {
-      _linalg_norm_vector_out(result, self, opt_num_ord, dim_, keepdim, opt_dtype);
+      Tensor result_ = at::linalg_vector_norm(self, opt_num_ord, opt_dim, keepdim, opt_dtype);
```

Comment:
Why doesn't this call linalg_vector_norm_out?

Reply:
`linalg_vector_norm_out` is not differentiable. In a future PR, I would like to change it to use `linalg_vector_norm_out` only when the out variant is called, and `linalg_vector_norm` otherwise